### DISCUSSION TITLE
Follow VSCode theme and/or force dark/light mode ?

### DISCUSSION THREAD

**Discussion by stevebeauge** [estruyf/vscode-front-matter#614] [Q&A] [answered]
Hello,

I'm evaluating Front Matter to add GUI on top of my new blog. Seems promising.

However, the extension has its own theme, which is very eye aggressive on dark mode (contrast of white text on top of nearly dark background is exhausting). This is very personal feeling, I know there's conjunction of a lot of factors (screen, room ligths, ...). Don't take it as a critic :). I use a vscode dark theme, but with softer contrasts ([Bearded theme monokai stone](https://marketplace.visualstudio.com/items?itemName=BeardedBear.beardedtheme) if it matters).

![image](https://github.com/estruyf/vscode-front-matter/assets/184604/4ecc6004-dab8-4584-b2ff-b6c51657a9ba)

Is it possible to let the extension follow the VS code theme, or to force light theme ?

Fun fact: the chatbot on front matter site suggest setting a setting that does not exists:

![image](https://github.com/estruyf/vscode-front-matter/assets/184604/98e74795-b12d-40f4-a278-dfff7ebaa551)

thanks !

**Comment by estruyf** [comment, marked as answer]
Hi @stevebeauge, in the upcoming version, we use by default the VS Code theme colours. In the current version, you can try it out by enabling the experimental features. More can be found here: https://frontmatter.codes/docs/experimental#visual-studio-code-theme-support

By the way, great to hear you are giving Front Matter a try!

**Comment by stevebeauge** [comment]
Perfect !

![image](https://github.com/estruyf/vscode-front-matter/assets/184604/a13a6b3a-04b9-4e5f-ab4a-32915dc3044c)

thanks for your great work !
